plugins/lazy-yt/config/settings.yml

```diff
@@ -0,0 +1,4 @@
+plugins:
+  lazy_yt_enabled:
+    default: true
+    client: false
```

Comment:
Can we make this a hidden setting? Since we're only going to use it temporarily and we will be moving to a differently named plugin soon, it's best not to show this setting to end users.

Reply:
I agree. That would be better.